### PR TITLE
Missed adding the arquillian instructions to kitchensink

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: kitchensink/README.md [hunk 1]
```diff
@@ -67,10 +67,18 @@ Access the application
...
+This quickstart provides Arquillian tests. By default, these tests are configured to be skipped as Arquillian tests require the use of a container. 
+
+_NOTE: The following commands assume you have configured your Maven user settings. If you have not, you must include Maven setting arguments on the command line. See [Run the Arquillian Tests](../README.html/#arquilliantests) for complete instructions and additional options._
+
+1. Make sure you have started the JBoss Server as described above.
+2. Open a command line and navigate to the root directory of this quickstart.
+3. Type the following command to run the test goal with the following profile activated:
 
-This quickstart provides Arquillian tests. See [Run the Arquillian Tests](../README.html/#arquilliantests) for information on how to set up and run the tests. When asked to start the JBoss Server, this quickstart requires that you start it with the _web_ profile.
+                  mvn clean test -Parq-jbossas-remote
```
Comment (pmuir) [inline kitchensink/README.md]: OTT indent!!!!

8 spaces only!

Comment (sgilda) [inline kitchensink/README.md]: Right. I need to go back and fix a bunch. Copy/paste propagated the issue.

**Comment by sgilda** [comment]
Modified all of the files. Big changes to the jts-distributed-crash-rec and jta-crash-rec to conform to the new template and use instructions in the root README.

**Comment by sgilda** [comment]
Missing authors on the xml-\* quickstart. Squashed commits.

**Pull request merged**